master/internal/resourcemanagers/scheduler_test.go

```diff
@@ -110,6 +118,9 @@ func (t *mockTask) Receive(ctx *actor.Context) error {
 }
 
 type mockAgent struct {
+	onStartTaskContainer func(sproto.StartTaskContainer) error
```

Comment:
Why do we need this functions? They're not used anywhere, right?

Reply:
No. I will remove those functions.